src/cl/lighthouse/lighthouse_launcher.star

```diff
@@ -353,46 +353,58 @@ def get_beacon_config(
         "--metrics-allow-origin=*",
         "--metrics-port={0}".format(BEACON_METRICS_PORT_NUM),
         # ^^^^^^^^^^^^^^^^^^^ METRICS CONFIG ^^^^^^^^^^^^^^^^^^^^^
+        # "--allow-insecure-genesis-sync",
     ]
 
     if network not in constants.PUBLIC_NETWORKS:
         cmd.append("--testnet-dir=" + constants.GENESIS_CONFIG_MOUNT_PATH_ON_CONTAINER)
-    else:
-        cmd.append("--network=" + network)
-        cmd.append("--checkpoint-sync-url=" + constants.CHECKPOINT_SYNC_URL[network])
-
-    if network == "kurtosis":
-        if boot_cl_client_ctxs != None:
+        if network == "kurtosis":  # Kurtosis
+            if boot_cl_client_ctxs != None:
+                cmd.append(
+                    "--boot-nodes="
+                    + ",".join(
+                        [
+                            ctx.enr
+                            for ctx in boot_cl_client_ctxs[: constants.MAX_ENR_ENTRIES]
+                        ]
+                    )
+                )
+                cmd.append(
+                    "--trusted-peers="
+                    + ",".join(
+                        [
+                            ctx.peer_id
+                            for ctx in boot_cl_client_ctxs[: constants.MAX_ENR_ENTRIES]
+                        ]
+                    )
+                )
+        elif network == "ephemery":  # Ephemery
+            cmd.append(
+                "--checkpoint-sync-url=" + constants.CHECKPOINT_SYNC_URL[network]
+            )
             cmd.append(
                 "--boot-nodes="
-                + ",".join(
-                    [
-                        ctx.enr
-                        for ctx in boot_cl_client_ctxs[: constants.MAX_ENR_ENTRIES]
-                    ]
+                + shared_utils.get_devnet_enrs_list(
+                    plan, el_cl_genesis_data.files_artifact_uuid
                 )
             )
+        else:  # Devnets
+            # TODO Remove once checkpoint sync is working for verkle
+            if "verkle" not in network:
```

Comment:
similarly perhaps take verkle out as a constant

Reply:
its a temp workaround. Hopefully will be gone in a few weeks. 

also I dislike constants when they are obvious like this case.

Reply:
Same issue as the `kurtosis` string, we risk typos and renames are tough.

Reply:
https://github.com/kurtosis-tech/ethereum-package/pull/459/commits/906932d434d15e5ee0457d471af9a9e2c6e5c238 Fixed.